numpy/core/records.py

```diff
@@ -595,8 +595,30 @@ def _deprecate_shape_0_as_None(shape):
 
 def fromarrays(arrayList, dtype=None, shape=None, formats=None,
                names=None, titles=None, aligned=False, byteorder=None):
-    """ create a record array from a (flat) list of arrays
+    """Create a record array from a (flat) list of arrays
 
+    Parameters
+    ----------
+    arrayList : sequence
+        Iterable container of array-like objects (such as lists, tuples,
+        and ndarrays).
+    dtype : data-type, optional
+        valid dtype for all arrays
+    shape : int or tuple of ints, optional
+        Shape of the resulting array. If not provided, inferred from
+        ``arrayList[0]``.
+    formats, names, titles, aligned, byteorder :
+        If `dtype` is ``None``, these arguments are passed to
+        `numpy.format_parser` to construct a dtype. See that function for
+        detailed documentation.
+
+
+    Returns
+    -------
+    record array consisting of given arrayList columns.
```

Comment:
```suggestion
    np.recarray
        Record array consisting of given arrayList columns.
```
According to the [numpydoc formatting guide](https://numpydoc.readthedocs.io/en/latest/format.html), it's fine to not name the return variable, but the type is always required.